assignment.rb

```diff
@@ -0,0 +1,100 @@
+require "csv"
+require "./user.rb"
+
+### CSV set-up
+current_users = []
+csv = CSV.foreach('./users.csv', headers: true, header_converters: :symbol) { |row|
+  row.to_hash
+  user = User.new(row[:name], row[:pin], row[:balance])
+# Curious if there is a way to do this without creating an array.
+# All the User info is right here, I just don't know how to access it...
+# ... unless I push it to an array.
+  current_users.push(user)
```

Comment:
Given only 1 person can operate the machine at a time, I reckon you can only have 1 `current_user`, so you can instead have
```ruby
current_user = nil
# ...
current_user = User.new(...)
```